user-manual/preservation/preservation-planning.rst

```diff
@@ -70,8 +70,8 @@ When creating a new format version, the following fields are available:
   choose "14".
 
 * PRONOM ID - The specific format version's unique identifier in
-  `PRONOM <http://www.nationalarchives.gov.uk/PRONOM/Default.aspx>`_, the UK
-  National Archives's format registry. This is optional, but highly recommended.
+  `PRONOM <PRONOM_>`_, the UK National Archives's format registry. This is 
+  optional, but highly recommended.
   Many identification and characterization tools, use PRONOM ID's to recognize
```

Comment:
While you're in here can you fix this comma splice? :slightly_smiling_face:

Reply:
Done!